actors/Decorations/DEAD.dec

```diff
@@ -910,7 +910,7 @@ ACTOR HangingBody: Meat2 Replaces Meat2
 	+SHOOTABLE
     GibHealth 10
 	damagefactor "Fire", 0.0
-	DropItem "CLIP1Drop" 128
+	DropItem "NewClip" 128
```

Comment:
I would replace this with BulletCartridge. When I looked at BDv21's code, ClipP1Drop was the drop for the pistol clips, since Clip1 was the pistol ammo in BDv21. BulletCartridge would make sense here.

Reply:
Done